src/blockchains/eth_blocks/eth_blocks_worker.ts

```diff
@@ -0,0 +1,68 @@
+import { logger } from '../../lib/logger';
+import { constructRPCClient } from '../../lib/http_client';
+import { BaseWorker } from '../../lib/worker_base';
+import { Web3Interface, constructWeb3Wrapper, safeCastToNumber } from '../eth/lib/web3_wrapper';
+import { nextIntervalCalculator, analyzeWorkerContext, setWorkerSleepTime, NO_WORK_SLEEP } from '../eth/lib/next_interval_calculator';
+import { fetchBlocks } from '../eth/lib/fetch_data';
+import { ETHBlock } from '../eth/eth_types';
+import { HTTPClientInterface } from '../../types';
+
+
+export class ETHBlocksWorker extends BaseWorker {
```

Comment:
This is a migration of this [repo](https://github.com/santiment/eth-blocks-exporter/blob/master/index.js). The code should live here and old repo deprecated.